Automattic/simplenote-ios
# Making Lint Changes as Suggested by SwiftLint

Fixes #1022 

*Edit: * changing to use ~~Swift Package Manager~~ existing rake binary of SwiftLint in PR #1143 rather than using Cocoapods.

### Fix
Making changes suggested by SwiftLint.

### Screenshots
![Screen Shot 2021-02-09 at 4 42 23 PM](https://user-images.githubusercontent.com/13263478/107438466-1f074000-6af6-11eb-8760-6c7441a1a25d.png)

### Test
1. Build Simplenote
2. See that there are errors/warnings showing up in warnings pane on sidebar when there are SwiftLint errors.

### Review
Only one developer and one designer are required to review these changes, but anyone can perform the review.

**Note: I separated out the whitespace/etc changes from the NSLocalizedString comment changes in separate commits for easier reviewing.**

### Release
These changes do not require release notes.
